src/posix.mak

```diff
@@ -351,7 +351,7 @@ $G/backend.a: $(G_OBJS)
 $G/dmd_frontend: $(FRONT_SRCS) $D/gluelayer.d $(ROOT_SRCS) $G/newdelete.o $(STRING_IMPORT_FILES) $(HOST_DMD_PATH)
 	CC=$(HOST_CXX) $(HOST_DMD_RUN) -of$@ $(MODEL_FLAG) -vtls -J$G -J../res -L-lstdc++ $(DFLAGS) $(filter-out $(STRING_IMPORT_FILES) $(HOST_DMD_PATH),$^) -version=NoBackend
 
-dmd: $G/dmd
+dmd: $G/dmd $G/dmd.conf
```

Comment:
One reason this wasn't done previously is that a dmd.conf in the current working dir applies to any dmd being run, it has the highest precedence in the config search order. Super-annoying IMO, but I won't argue any more.
Since we've switched to explicit -conf= it shouldn't cause much troubles.